### ISSUE TITLE
async def is not supported

### ISSUE THREAD

**Issue by littleningmeng** [scottrogowski/code2flow#49]

**Comment by scottrogowski** [comment]
Hi @littleningmeng. Could you provide more context on what is not working or an example project?

**Comment by littleningmeng** [comment]
Hi @scottrogowski 
you can just write a class with async method like this
```python
class A:
    def __init__(self):
        pass
    async def test(self):
        pass
```
code2flow would not give the definition of `test` in the result graph.

**Comment by scottrogowski** [comment]
If I am understanding everything correctly, it was an easy fix. I haven't used async/await much professionaly though so I'm not sure whether I'm missing some complexity here. Could you take a look? https://github.com/scottrogowski/code2flow/pull/57

**Comment by littleningmeng** [comment]
> If I am understanding everything correctly, it was an easy fix. I haven't used async/await much professionaly though so I'm not sure whether I'm missing some complexity here. Could you take a look? #57

awesome, it should work

**Comment by scottrogowski** [comment]
Just released as v2.5. Closing. Let me know if you have any issues
